qiskit-superstaq/qiskit_superstaq/superstaq_backend.py

```diff
@@ -35,21 +35,26 @@ def __init__(self, provider: qss.SuperstaqProvider, target: str) -> None:
             target: A string containing the name of a target backend.
         """
         self._provider = provider
+
+        target_info = self._provider._client.target_info(target)["target_info"]
         self.configuration_dict = {
             "backend_name": target,
             "backend_version": "n/a",
-            "n_qubits": -1,
-            "basis_gates": None,
+            "n_qubits": target_info.get("num_qubits"),
+            "basis_gates": target_info.get("native_gate_set"),
             "gates": [],
             "local": False,
             "simulator": False,
             "conditional": False,
             "open_pulse": False,
             "memory": False,
-            "max_shots": -1,
+            "max_shots": None,
             "coupling_map": None,
         }
+        target_info.pop("num_qubits", None)
+        target_info.pop("native_gate_set", None)
```

Comment:
Should there also be: 
``` 
target_info.pop("target", None)
```
given that `"backend_name": target` is present in the dict, or do we later want to update and list both the 'superstaq name' and 'provider name' -- e.g. "qtm_h1-1_qpu" and "Quantinuum H1-1"

Reply:
Oops. I forgot to pop this one. Done.